.readthedocs.yaml

```diff
@@ -0,0 +1,23 @@
+# Read the Docs configuration file for Sphinx projects
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+# Required
+version: 2
+
+# Set the OS, Python version and other tools you might need
+build:
+  os: ubuntu-22.04
+  tools:
+    python: "3.11"
```

Comment:
I went a bit conservative with `3.11` instead of `3.12`, as I did not have a good environment to test it. As long as the pinned version of `sphinx` in `docs/requirements.txt` works, it should be safe to bump.